charts/stable/authentik/templates/common.yaml

```diff
@@ -1,11 +1,17 @@
 {{/* Make sure all variables are set properly */}}
 {{- include "tc.v1.common.loader.init" . }}
 
-{{/* Render secret */}}
-{{- include "authentik.secret" . }}
+{{/* Render secrets for authentik */}}
+{{- $secrets := include "authentik.secret" . | fromYaml -}}
+{{- if $secrets -}}
+{{- $_ := set .Values.secret "" $secrets -}}
```

Comment:
You need to fix this, you're missing the name in a lot of places and instead have `"`

Also please check your previous apps also noticed it there.